fml/synchronization/waitable_event_unittest.cc

```diff
@@ -31,7 +31,7 @@ void SleepFor(TimeDelta duration) {
 
 void EpsilonRandomSleep() {
   TimeDelta duration =
-      TimeDelta::FromMilliseconds(static_cast<unsigned>(rand()) % 20u);
+      TimeDelta::FromMilliseconds(static_cast<unsigned>(arc4random()) % 20u);
```

Comment:
Doesn't matter for these unit tests, but can't hurt.
```
error: Function 'rand' is obsolete because it implements a poor random number generator.  Use 'arc4random' instead [clang-analyzer-security.insecureAPI.rand,-warnings-as-errors]
```